.gitignore

```diff
@@ -64,3 +64,6 @@ typings/
 .DS_Store
 
 .vscode/
+
+# Jetbrains project settings
+.idea/
```

Comment:
nit: newline 🙃

Reply:
would it make sense to define an [EditorConfig](https://editorconfig.org/) file? (If so, I can open a new issue).

Reply:
@Ryuno-Ki There's already one.